committee/committee.go

```diff
@@ -202,6 +202,8 @@ func (ec *committee) Start(ctx context.Context) (err error) {
 		return err
 	}
 	if ec.latestHeightInArchive() >= ec.ceilingHeight {
+		zap.L().Info("stop syncing")
+		ec.terminated = true
```

Comment:
nit terminated -> terminatedCarrier

Reply:
does not hurt to close(ec.terminate) here
otherwise after Stop(), it becomes a dangling channel